graph/src/schema/input_schema.rs

```diff
@@ -282,3 +291,61 @@ impl Inner {
         Entity::try_make(self.pool.clone(), iter)
     }
 }
+
+/// Create a new pool that contains the names of all the types defined
+/// in the document and the names of all their fields
+fn atom_pool(document: &s::Document) -> AtomPool {
+    let mut pool = AtomPool::new();
+    // These two entries are always required
```

Comment:
three entries.

Reply:
Heh .. two of them shouldn't be there. I'll have a follow-on PR that removes `__typename` and `g$parent_id`; they are only for the query side.